frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -17,7 +17,7 @@ import { router, urlToAction } from 'kea-router'
 import api, { ApiMethodOptions, getJSONOrNull } from 'lib/api'
 import { DashboardPrivilegeLevel, OrganizationMembershipLevel } from 'lib/constants'
 import { Dayjs, dayjs, now } from 'lib/dayjs'
-import { captureTimeToSeeData, currentSessionId, TimeToSeeDataPayload } from 'lib/internalMetrics'
```

Comment:
Is it safe to delete this metric?

Reply:
https://posthog.slack.com/archives/C076R4753Q8/p1724345168785269

Yep, currently we're sending a lot of requests to the `/insight/timing` endpoint but not really using the metrics there + we've commented out the clickhouse table creation script. It's better to have it in PostHog itself (over metabase which is going away) so we can view it there + add alerting